src/components/PageSizer.js

```diff
@@ -1,23 +1,29 @@
 import React, { useCallback } from 'react'
 import propTypes from 'prop-types'
+import { useManifest } from 'use-manifest'
+
 import useUrlParamState from '../hooks/useUrlParamState'
 
-const DEFAULT_PAGE_SIZES = [10, 20, 50, 100, 200]
+const DEFAULT_PAGE_SIZES = [10, 20, 50, 100]
 
 const DEFAULT_TEXT_GENERATOR = size => `Show ${size} entries`
 
+const numbersAscending = (a, b) => a - b
+
 const PageSizer = ({ pageSizes = DEFAULT_PAGE_SIZES, pageSizeLabelGenerator = DEFAULT_TEXT_GENERATOR }) => {
   const [urlState, updateUrl] = useUrlParamState()
-  const { pageSize } = urlState
+  const { pageSize } = useManifest()
 
   const handlePageSizeChange = useCallback(ev => {
     const updatedPageSize = +ev.target.value
     updateUrl({ ...urlState, pageSize: updatedPageSize, page: 0 })
   }, [updateUrl, urlState])
 
+  if (!pageSizes.includes(pageSize)) pageSizes.push(pageSize)
+
   return (
     <select className='form-control' value={pageSize} onChange={handlePageSizeChange}>
-      {pageSizes.map(size => <option key={size} value={size}>{pageSizeLabelGenerator(size)}</option>)}
+      {pageSizes.sort(numbersAscending).map(size => <option key={size} value={size}>{pageSizeLabelGenerator(size)}</option>)}
```

Comment:
Unfortunately sort mutates the array, so the way this is currently structured is misleading. We should clone the array before calling sort and consider doing the sort on a different line with the return value not used so the mutation is obvious.